### ISSUE TITLE
Further support of old SGF

### ISSUE THREAD

**Issue by kaorahi** [sanderland/katrain#424]
In addition to #140, some old SGF files (IGS in 1995) end like this:

~~~
;B[fp]BL[500];

)
~~~

How about supporting them by replacing `;` with `;\s*` in `SGFPROP_PAT`? IGS (Internet Go Server) was the most popular place to play Go on the Internet in those days.

~~~
--- a/katrain/core/sgf_parser.py
+++ b/katrain/core/sgf_parser.py
@@ -392,7 +392,7 @@ class SGF:
 
     _NODE_CLASS = SGFNode  # Class used for SGF Nodes, can change this to something that inherits from SGFNode
     # https://xkcd.com/1171/
-    SGFPROP_PAT = re.compile(r"\s*(?:\(|\)|;|(\w+)((\s*\[([^\]\\]|\\.)*\])+))", flags=re.DOTALL)
+    SGFPROP_PAT = re.compile(r"\s*(?:\(|\)|;\s*|(\w+)((\s*\[([^\]\\]|\\.)*\])+))", flags=re.DOTALL)
     SGF_PAT = re.compile(r"\(;.*\)", flags=re.DOTALL)
 
     @classmethod
~~~

**Comment by sanderland** [comment]
do you have an example file? What happens?

**Comment by kaorahi** [comment]
This SGF causes a wrong white "1" after the correct black "1" on 9d1c500.

~~~
(;
SZ[19]TM[600]KM[0.500000]LT[]

;B[fp]BL[500];

)
~~~
![IGS_SGF](https://user-images.githubusercontent.com/38910552/117815404-668d9b80-b2a0-11eb-92b2-f18c5932d2ee.png)

**Comment by sanderland** [comment]
fixed in e974283
